frontend/src/components/SeasonForms/AreaSeasonForm.jsx

```diff
@@ -436,40 +448,66 @@ export default function AreaSeasonForm({
 
   return (
     <>
-      <FormContainer>
-        <div className="row">
-          {/* Area-level dates */}
-          {areaDateTypes.map((dateType) => (
-            <div key={dateType.name} className="col-lg-6 mb-4">
-              <h6 className="fw-normal">
-                {dateType.name}{" "}
-                <TooltipWrapper placement="top" content={dateType.description}>
-                  <FontAwesomeIcon icon={faCircleInfo} />
-                </TooltipWrapper>
-              </h6>
-
-              <PreviousDates
-                dateRanges={previousAreaDatesByType?.[dateType.name]}
-              />
-
-              <DateRangeFields
-                dateableId={parkArea.dateableId}
-                dateType={dateType}
-                dateRanges={areaDatesByType[dateType.name] ?? []}
-                updateDateRange={updateAreaDateRange}
-                addDateRange={addAreaDateRange}
-                removeDateRange={removeAreaDateRange}
-                dateRangeAnnuals={dateRangeAnnuals}
-                updateDateRangeAnnual={updateDateRangeAnnual}
+      {showBcpResSection && (
+        <FormContainer>
+          <div className="row">
+            {/* Area-level dates */}
+            {areaDateTypes.map((dateType) => (
+              <div key={dateType.name} className="col-lg-6 mb-4">
```

Comment:
I believe it won't be rendered, but would you like to keep it here? Since we don't create any dates at the Park Area level.